GoogleUtilities/AppDelegateSwizzler/GULAppDelegateSwizzler.m

```diff
@@ -284,6 +284,10 @@ + (void)unregisterAppDelegateInterceptorWithID:(GULAppDelegateInterceptorID)inte
 }
 
 + (void)proxyOriginalDelegate {
```

Comment:
I suspect it's the definition of this method with NS_EXTENSION_UNAVAILABLE, so for me as long as this method exists in messaging (even not calling it if it's extension) the compiler will fail.

Reply:
I update definition of the method in .h by removing `NS_EXTENSION_UNAVAILABLE` as well, so it should be fine.

Reply:
I added a test extension target to make sure we fail PR tests if Messaging becomes incompatible with extensions.